nifi-nar-bundles/nifi-salesforce-bundle/nifi-salesforce-processors/src/main/java/org/apache/nifi/processors/salesforce/QuerySalesforceObject.java

```diff
@@ -102,11 +107,11 @@
 @CapabilityDescription("Retrieves records from a Salesforce sObject. Users can add arbitrary filter conditions by setting the 'Custom WHERE Condition' property."
         + " The processor can also run a custom query, although record processing is not supported in that case."
         + " Supports incremental retrieval: users can define a field in the 'Age Field' property that will be used to determine when the record was created."
-        + " When this property is set the processor will retrieve new records. It's also possible to define an initial cutoff value for the age, filtering out all older records"
+        + " When this property is set the processor will retrieve new records. Incremental loading and record-based processing are only supported in property-based queries."
```

Comment:
It'd be useful to separate the "Property based query" and "Custom query" mode in the processor description.
E.g. 'Age Field' does not exist in "Custom query" mode.

Reply:
This is mentioned in the processor description here "Incremental loading and record-based processing are only supported in property-based queries"